code/game/machinery/adv_med.dm

```diff
@@ -456,6 +457,11 @@
 			robot = "Prosthetic:"
 		if(e.open)
 			open = "Open:"
+		if(e.min_broken_damage != initial(e.min_broken_damage))
+			var/increase = e.min_broken_damage - initial(e.min_broken_damage)
+			var/percent_increase = round(increase / initial(e.min_broken_damage) * 100)
+			var/sign = percent_increase > 0 ? "+" : ""
```

Comment:
what about a possibility for bone weakening?

Reply:
Seems to me like this should work?

Reply:
Not quite. Say you've got a e.min_broken_damage of 5, and its initial 15. It'd show 67 percent, when it should show 33. Maybe change the "" to "-"?